sqlglot/dataframe/sql/session.py

```diff
@@ -12,6 +12,8 @@
 from sqlglot.dataframe.sql.types import StructType
 from sqlglot.dataframe.sql.util import get_column_mapping_from_schema_input
 from sqlglot.helper import classproperty
+from sqlglot.optimizer import optimize as optimize_func
```

Comment:
We can now omit the alias here, since there's no `optimize` arg shadowing the import anymore.